src/components/Form.js

```diff
@@ -135,8 +135,34 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const foundHtmlTagIndex = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matchedHtmlTags  = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+
+                let isMatch = CONST.WHITELISTED_TAGS.some(r => r.test(inputValue));
```

Comment:
Can you bring this above line 150, given it is used either in the if block or after it?

Reply:
no because that would break the check for foundHtmlTagIndex, it should be defined prior to the if block.

Reply:
I am still confused on why can't we bring this above to `if(matchedHtmlTags)` part?

Reply:
because line 146 requires foundHtmlTagIndex to have run the VALIDATE_FOR_HTML_TAG_REGEX  prior to line 150, so that it can be checked in the if block. Moving to line 150 would cause problems in the if block.

Reply:
Sorry, can you share your slack profile name so I can discuss this 1:1.

Reply:
@techievivek I really appreciate the slack invitation but given that slack discussions can get quite lengthy, it is better to quickly resolve any confusion in the PR. 
You were asking if Line 139 to 142 can be put above line 150 if(matchedHtmlTags)
I can go through the 3 lines,
Line 139 foundHtmlTagIndex is being used in the if block and for it to check for index the regex has to have been run prior to the if block therefore it cannot be moved.
Line 140 matchedHtmlTags finds matches for VALIDATE_FOR_HTML_TAG_REGEX. It may not have any match at all. 
Line 142 isMatch depends on Line 140 whether there are any matches, it may or may not be updated 
both line 140 and 142 have to be set prior to the if condition check if (matchedHtmlTags). 
The concern I have is the code above is already tested and passes, moving code blocks may affect testing scenarios so it better to leave it as is. 
@robertKozik please provide second opinion on whether the code should be moved above line 150, at this point I prefer to leave it as is because it has been already tested in several platforms.

Reply:
@techievivek @sourcecodedeveloper I cannot see the downside for bringing the `isMatch` and `matchHtmlTags` variable initialisation to 148 line, the line right before `if (matchedHtmlTags) {` line. `isMatch` variable is not used before, its only used inside the if statement or later in line 160 in next if statement. Same as `matchHtmlTags`, it's used only inside the if statment.

`foundHtmlTagIndex` cannot be moved as its used inside the check in line 144.

Also I cannot agree with the statement that we shouldn't move anything because it's already tested. Our goal should be to ship high quality code, even if we will have to test it once more

Reply:
Maybe it was not clear but I only wanted to move the `isMatch` declaration and initialization before the if block.